lumicks/pylake/kymo.py

```diff
@@ -99,11 +154,19 @@ def _plot(self, image, **kwargs):
             aspect=(image.shape[0] / image.shape[1]) * (duration / width_um)
         )
 
+        x_lims = fetch_custom_property(kwargs, "xlim")
+        y_lims = fetch_custom_property(kwargs, "ylim")
+
```

Comment:
I would do the extracting of custom arguments before preparing the default arguments.

Reply:
Alright. I swapped it around. Just out of curiosity, why are custom ones typically handled first?

Reply:
I was too vague here.

The motivation was to bring the usage and the declaration of `extract_custom_properties` together not specifically to move it before `default_kwargs`. 

Usually is good to strive for minimizing the mental workload of reading code, so declarations should happen only when needed. But putting the function between the other arguments (`width_um`, `duration` and `default_kwargs`) and their usage would be also confusing, so the best solution would be to move the custom properties to after the declaration.

Reply:
Ah okay. Makes sense yes. With the new one liner with default argument you suggested I would say this block is nicely cleaned up now :)